cmd/csi-addons/reclaimspace.go

```diff
@@ -83,7 +84,17 @@ func (nrs *NodeReclaimSpace) Init(c *command) error {
 	if c.stagingPath == "" {
 		return fmt.Errorf("stagingpath is not set")
 	}
-	nrs.stagingTargetPath = fmt.Sprintf("%s/%s/globalmount", c.stagingPath, c.persistentVolume)
+
+	if !c.legacy {
```

Comment:
we could, but the command does not use any Kubernetes client at all, it is intended to be functional outside of Kubernetes too.

Reply:
Okay, 

If it can run seamlessly on k8s version less than 1.24 than I am fine. 

Or do we need to tweak legacy variable depending on the k8s version too?

(This change will need to go into rook)